Tribler/Core/Libtorrent/LibtorrentMgr.py

```diff
@@ -517,13 +517,15 @@ def start_download_from_uri(self, uri):
 
         return None
 
+    @blocking_call_on_reactor_thread
```

Comment:
This was for backwards compatibility to the GUI 8528a5389d2e17f3193bbfc1b434d16f54b32a8b